src/index.js

```diff
@@ -1,91 +1,15 @@
 /* eslint-disable import/extensions */
 /* eslint-disable no-loop-func */
```

Comment:
Please, don't disable linter checks. Linter checks are placed in order to improve the code quality of students. Kindly remove the linter disable rules. I would recommend fixing the linter errors instead of disabling the linter checks.